source/materials/OpticalMaterialProperties.h

```diff
@@ -83,6 +83,10 @@ namespace opticalprops {
 
   G4MaterialPropertiesTable* PMMA();
 
+  G4MaterialPropertiesTable* Copper();
```

Comment:
Since you left out the steel and copper optical properties from this PR, I would remove also these remaining modifications to the `OpticalMaterialProperties` files.